webapp/components/generic/SearchableInput/SearchableInput.spec.js

```diff
@@ -117,5 +110,15 @@ describe('SearchableInput.vue', () => {
         expect(mocks.$router.push).toHaveBeenCalledWith('?hashtag=Hashtag')
       })
     })
+
+    it('opens the search result page when enter is pressed', async () => {
```

Comment:
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-05-13T17:55:48Z" title="Wednesday, May 13th 2020, 7:55:48 pm +02:00">May 13, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -117,5 +110,15 @@ describe('SearchableInput.vue', () => {
         expect(mocks.$router.push).toHaveBeenCalledWith('?hashtag=Hashtag')
       })
     })
+
+    it('searches for the term when enter is pressed', async () => {
```


Please update the text. It is the old one and very misleading